### PR TITLE
support 'qsub -d' in Slurm submission wrapper

### PR DESCRIPTION
**Pull request by boegel**
`qsub -d` allows to specify a specific starting directory (default remains `$HOME`)

### CODE REVIEW

File: files/qsub.pl [hunk 1]
```diff
@@ -177,6 +177,7 @@ sub make_command
         'A=s'      => \$account,
         'b=s'      => \$wrap,
         'cwd'      => sub { }, # this is the default
+        'd=s'      => \$workdir,
```
Comment (stdweird) [inline files/qsub.pl]: you mean like the `wd` option?

File: files/qsub.pl [hunk 2]
```diff
@@ -349,11 +347,16 @@ sub make_command
     push(@command, "--ntasks=$node_opts->{task_cnt}") if $node_opts->{task_cnt};
     push(@command, "--nodelist=$node_opts->{hostlist}") if $node_opts->{hostlist};
 
-    push(@command, "--chdir=$workdir") if $workdir;
-
     push(@command, "--mincpus=$res_opts->{ncpus}") if $res_opts->{ncpus};
     push(@command, "--ntasks-per-node=$res_opts->{mppnppn}")  if $res_opts->{mppnppn};
 
+    if ($workdir) {
```
Comment (stdweird) [inline files/qsub.pl]: why is this needed? what was not working with the old code?

File: files/qsub.pl (hunk 1 above)
Comment (boegel) [reply files/qsub.pl]: `qsub -wd` doesn't exist, I don't know why `wd` is even an option

Comment (stdweird) [reply files/qsub.pl]: in particular, if the `wd` doesn't exist for torque, just rename it? else, use `wd|d=s` to set a 2nd optionname

File: files/qsub.pl (hunk 2 above)
Comment (boegel) [reply files/qsub.pl]: Without this, `--chdir=$HOME` is always used, which is wrong if an alternate location is passed with `qsub -d <path>`

**Comment by stdweird** [comment]
why didn't you remove the `wd` option?

**Comment by boegel** [comment]
To not break backward compatibility, maybe people started using it...
